test/vaults/integration/BalanceSheet.t.sol

```diff
@@ -181,39 +181,6 @@ contract BalanceSheetTest is BaseTest {
         assertEq(erc20.balanceOf(address(this)), 0);
     }
 
-    function testNoteDeposit() public {
```

Comment:
Nit: Next week, we should add back an updated version of this as we still have `noteDeposit`. Definitely not necessary for this PR.

Reply:
Added back to https://github.com/centrifuge/protocol-v3/pull/306